src/view/Page.tsx

```diff
@@ -0,0 +1,69 @@
+import React, {ReactElement, useState} from "react";
+import {Form} from "./Form";
+import {ItemCard} from "./ItemCard";
+import LocalStorageService from "../service/LocalStorageService";
+import TodoLine from "../model/TodoLine";
+
+export const Page = () => {
+
+    const VIEW_LIST_MODE = "list"
+    const VIEW_FORM_MODE = "form"
+
+    const [item, setItem] = useState<TodoLine>()
+    const [view, setView] = useState("list")
+    const [count, forceRefresh] = useState(0)
+
+    function getView() {
+        if (view === VIEW_LIST_MODE) {
+            return getListView()
+        } else if (item!=null) {
+            return <Form item={item} goToListView={()=>setView(VIEW_LIST_MODE)} />
+        } else {
+            return <Form item={undefined} goToListView={()=>setView(VIEW_LIST_MODE)} />
+        }
```

Comment:
En react, il est plus standart de mettre la logique directement dans le return a l'aide de guardian. Meme si ici, comme il y a 3 cas, la question se pose, le if else est peut etre plus claire

```suggestion
   return (
    <>
      {view === ViewMode.LIST && getListView()}
      {view !== ViewMode.LIST &&
        (item !== null ? (
          <Form item={item} goToListView={() => setView(VIEW_LIST_MODE)} />
        ) : (
          <Form item={undefined} goToListView={() => setView(VIEW_LIST_MODE)} />
        ))}
    </>
  );
```

Reply:
Pour un IF ELSE pourquoi pas sinon je trouve cette expression moins lisible.